### PR TITLE
feat: #571 guidelines improvement

### PR DESCRIPTION
**Pull request by Anok666**
https://github.com/cyfronet-fid/eosc-search-service/issues/571

### CODE REVIEW

**Comment by aleksandrahojszyk** [comment]
Screenshot 2023-05-08 at 11 48 56
Headliners -> semibold weight 600 
Screenshot 2023-05-08 at 11 50 39
odległość między boxami -> 24px

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
If we don't have data we should show "-"
image

**Comment by Marcelinna** [comment]
why dla guidelines cyfronet I see Identifier "-" ?
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
Add "-" when provider by is empty
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
key is shown, not value
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
redundant dash
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
I have strange value
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
I have a strange value
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
I have a strange value 
image

**Comment by Marcelinna** [comment]
- [ ] QA
- [ ] DEV
Please delete "s"
image

**Comment by Michal-Kolomanski** [comment]
Those links should be clickable, right? @aleksandrahojszyk @Marcelinna 
image

Only Related Standard URI is clickable now.

File: ui/apps/ui/src/app/app.routing.module.ts [hunk 1]
```diff
@@ -27,10 +27,10 @@ import { PrivacyPolicyModule } from '@pages/privacy-policy/privacy-policy.module
         loadChildren: () => TrainingsPageModule,
       },
       // TODO: temporarily off
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/app.routing.module.ts:29]: Remove comment

**Review by Michal-Kolomanski — COMMENTED** [review]
This should be probably mapped into something. @Marcelinna @agpul @aleksandrahojszyk help Artur on that :D 
image

File: ui/apps/ui/src/app/app.routing.module.ts (hunk 1 above)
Comment (Anok666) [reply ui/apps/ui/src/app/app.routing.module.ts]: done

File: ui/apps/ui/src/app/collections/data/index.ts [hunk 2]
```diff
@@ -56,7 +56,7 @@ export const ADAPTERS: IAdapter[] = [
   dataSourcesAdapter,
   trainingsAdapter,
   // TODO: temporarily off
```
[the remark this answers is not in the document]
Comment (Anok666) [reply ui/apps/ui/src/app/collections/data/index.ts]: done

**Review by Michal-Kolomanski — APPROVED** [review]

**Review by Michal-Kolomanski — CHANGES_REQUESTED** [review]
Remove recommendations request in IG tab

image

**Review by Marcelinna — APPROVED** [review]

File: ui/apps/ui/src/app/components/recommendations/recommendations.component.ts [hunk 3]
```diff
@@ -33,11 +33,15 @@ export class RecommendationsComponent implements OnInit {
     this._customRoute.collection$
       .pipe(
         untilDestroyed(this),
-        switchMap((panelId) =>
-          this._recommendationsService
+        switchMap((panelId) => {
+          // TODO: block request if interoperability guidelines
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/recommendations/recommendations.component.ts:37]: Personally, I would not use "TODO" here. 
Maybe something like: "IG recommendations are not supported" or no comment would be better?

**Review by Michal-Kolomanski — APPROVED** [review]

File: ui/apps/ui/src/app/components/recommendations/recommendations.component.ts (hunk 3 above)
Comment (Anok666) [reply ui/apps/ui/src/app/components/recommendations/recommendations.component.ts]: done, 
I thought so

**Pull request merged**